app/gui/src/project-view/components/GraphEditor/GraphVisualization/visualizationData.ts

```diff
@@ -96,12 +96,7 @@ export function useVisualizationData({
   )
 
   const executeExpression = async (
-    visulizationModule: string,
-    expressionString: string,
-    formatFunction:
-      | ((arg: any, tempModule: Ast.MutableModule) => Ast.Owned<Ast.MutableExpression>)
-      | null,
-    ...positionalArgumentsExpressions: any[]
+    expressionFunction: (nodeIdentifier: string) => Ast.Owned<Ast.MutableOprApp>,
```

Comment:
Does the return type need to be this specific? I would think it could be any `Ast.Owned<Ast.Expression>`